src/graphql/entries.ts

```diff
@@ -77,18 +78,76 @@ export async function updateEntry({
   return copyOfDataFromPreviewApp;
 }
 
-function updateRichTextField({
+interface RichTextLink {
+  block: (Entity & CollectionItem)[];
+  inline: (Entity & CollectionItem)[];
+}
+
+function isEntityLinkEmpty(obj: RichTextLink) {
+  return Object.values(obj).every((arr) => arr.length === 0);
+}
+
+async function updateRichTextField({
```

Comment:
can we also call it there now?
https://github.com/contentful/live-preview/pull/139/files#diff-0b6094972cd316ac3e76d7602561d8d08032a3e09d809cbf03e8574cc91b3062R198
Otherwise, it will not work for nested references

Reply:
Done 👍